build/Dockerfile.distroless

```diff
@@ -0,0 +1,226 @@
+# This is our strategy for getting a clean (from CVE perspective) base image for python-based services.
```

Comment:
Might be worth removing this comment as it deviates from the python strategy a bit.

Alternatively would having this be `FROM public.ecr.aws/j9h1x6x3/distroless-python:3.11-deb12u7` make sense? I see python is installed later on but figure the specifics of that installation are not compatible with our distroless-python base image.

Reply:
Yeah I debated this but figured I didn't want to mess with the original dockerfile contents too much pre-distroless stage. It could work but I think we would need to do it after the go stage and before we install unoconverter.